# `plone:4.3.18-alpine`

## Docker Metadata

- Image ID: `sha256:5062fa130c75c68c7363f5278b29d4e7f37bfdba8425b5c37187c567914fec2b`
- Created: `2019-01-31T03:06:22.29051468Z`
- Virtual Size: ~ 362.97 Mb  
  (total size of all layers on-disk)
- Arch: `linux`/`amd64`
- Entrypoint: `["/docker-entrypoint.sh"]`
- Command: `["start"]`
- Environment:
  - `PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG=C.UTF-8`
  - `PYTHONIOENCODING=UTF-8`
  - `GPG_KEY=C01E1CAD5EA2C4F0B8E3571504C367C218ADD4FF`
  - `PYTHON_VERSION=2.7.15`
  - `PYTHON_PIP_VERSION=19.0.1`
  - `PIP=9.0.3`
  - `ZC_BUILDOUT=2.11.4`
  - `SETUPTOOLS=39.1.0`
  - `WHEEL=0.31.1`
  - `PLONE_MAJOR=4.3`
  - `PLONE_VERSION=4.3.18`
  - `PLONE_MD5=c9932e919254a2799b0e340b2ecbda9b`
